demos/browser/app/meetingV2/meetingV2.html

```diff
@@ -4,6 +4,16 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
   <link rel="shortcut icon" href="data:image/x-icon;," type="image/x-icon">
+  <script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs@2.7.0/dist/tf.min.js" 
```

Comment:
I would really like a plan for making this conditional/lazy, probably by dynamically adding the `script` elements when activating the relevant filter.

Do you need this with the filters that are present in the PR at this point?

Reply:
segmentation filters are removed. But good callout, lazy loading seems easy to implement and I will do that when the filter is put back in.